tests/Makefile

```
@@ -0,0 +1,32 @@
+ORG=libhkl
+REPO=tests
+LOGFILE=build-log.txt
+TAG=latest
+LOG_DIR=.
+
+FULLNAME=$(ORG)/$(REPO):$(TAG)
+
+build ::
+	echo "# started: " `date --iso-8601="seconds"` | tee $(LOGFILE)
```

`--iso-8601` is not an option on all systems; `-I` is more widely available.

```suggestion
	echo "# started: " `date -I"seconds"` | tee $(LOGFILE)
```

---

I recommend to apply this same suggestion to the builder/Makefile.  I could not easily add that through the GH Review web UI.